### PR TITLE
update some freedesktop metadata files and install them when appropriate

### PR DESCRIPTION
**Pull request by rohanlean**
#### Summary
Build "freedesktop metadata files are updated and installed when appropriate"

#### Purpose of change

Some of the freedesktop metadata files were outdated, had non-standard names, and were not installed when appropriate by the Makefile.

#### Describe the solution

I have updated and renamed the files, added installation instructions to the Makefile where appropriate, and updated the flatpak manifest.  (The flatpak manifest expectantly refers to the 0.F branch now, because it relies on the new Makefile.)

#### Testing

I have successfully built and installed the flatpak.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1016,6 +1016,9 @@ install: version $(TARGET)
 	cp -R --no-preserve=ownership data/help $(DATA_PREFIX)
 ifeq ($(TILES), 1)
 	cp -R --no-preserve=ownership gfx $(DATA_PREFIX)
+	install -Dm755 -t /app/share/applications/ data/xdg/org.cataclysmdda.CataclysmDDA.desktop
```
Comment (ZhilkinSerg) [inline Makefile:1019]: What if you are not using Flatpak?

Comment (rohanlean) [reply Makefile:1019]: These files are not flatpak-specific.  You would want them in any freedesktop system.

Comment (rohanlean) [reply Makefile:1019]: Oh, I understand now.  Sorry -- I had just copied the files from the flatpak manifest and did not make the installation path variable.  I will push a fix

**Comment by actual-nh** [comment]
Of course, the version description text in appdata.xml will need updating for 0.F once all that is finalized.

**Pull request merged**